enarx-keep-sgx-shim/src/handler.rs

```diff
@@ -263,10 +267,16 @@ impl<'a> Handler<'a> {
         }
 
         let cursor = self.block.cursor();
-        let _: &mut [libc::iovec] = unsafe { cursor.alloc(trusted.len()).or(Err(libc::EMSGSIZE))? };
+        let (mut cursor, _) = unsafe {
```

Comment:
```suggestion
        let (mut c, _) = unsafe {
```
Let's just name cursor `c` in this function. This will resolve multiple instances of jankyness.